finish/traffic.yaml

```diff
@@ -24,19 +24,11 @@ spec:
   gateways:
   - sys-app-gateway
   http:
-  # tag::route[]
   - route:
     - destination:
         port:
           number: 9081
         host: inventory-service
-    # tag::istioRetry[]
     retries:
-      # tag::attempts[]
       attempts: 4
-      # end::attempts[]
-      # tag::retryOn[]
-      retryOn: 5xx
-      # end::retryOn[]
-    # end::istioRetry[]
```

Comment:
Since we removed `istioRetry[]` tag, the retry section is always displayed in the code pane, even before the users are asked to add that retry section.

Reply:
That's a good point. I think I can add back the istioRetry tag and still fix the blank line issue